src/status_im2/contexts/chat/menus/pinned_messages/view.cljs

```diff
@@ -14,7 +14,6 @@
 
 (defn message-render-fn
   [{:keys [deleted? deleted-for-me?] :as message} _ _ context]
-  ;; TODO (flexsurfer) probably we don't want reactions here
```

Comment:
this can't be just removed, its not addressed yet, if we remove this, we need to hide reactions